playbooks/install_monitoring.yml

```diff
@@ -29,12 +29,3 @@
   roles:
     - role: mirsg.infrastructure.monitoring_client
     - role: mirsg.infrastructure.firewalld
-      vars:
```

Comment:
Am now setting these at the inventory level instead.

Reply:
nice, makes sense to put them in the inventory